### ISSUE TITLE
Error with traceback.print_exception(*err) in Python 3

### ISSUE THREAD

**Issue by hwkns** [flavioamieiro/nose-ipdb#16]
```
Traceback (most recent call last):
  File "/home/vagrant/.py3/lib/python3.4/site-packages/nose/case.py", line 134, in run
    self.runTest(result)
  File "/home/vagrant/.py3/lib/python3.4/site-packages/nose/case.py", line 152, in runTest
    test(result)
  File "/home/vagrant/.py3/lib/python3.4/site-packages/django/test/testcases.py", line 184, in __call__
    super(SimpleTestCase, self).__call__(result)
  File "/home/vagrant/.py3/lib/python3.4/site-packages/unittest2/case.py", line 410, in __call__
    return self.run(*args, **kwds)
  File "/home/vagrant/.py3/lib/python3.4/site-packages/unittest2/case.py", line 354, in run
    result.addFailure(self, sys.exc_info())
  File "/home/vagrant/.py3/lib/python3.4/site-packages/nose/proxy.py", line 146, in addFailure
    plugins.addFailure(self.test, err)
  File "/home/vagrant/.py3/lib/python3.4/site-packages/nose/plugins/manager.py", line 99, in __call__
    return self.call(*arg, **kw)
  File "/home/vagrant/.py3/lib/python3.4/site-packages/nose/plugins/manager.py", line 167, in simple
    result = meth(*arg, **kw)
  File "/home/vagrant/.py3/lib/python3.4/site-packages/ipdbplugin.py", line 54, in addFailure
    self.debug(err)
  File "/home/vagrant/.py3/lib/python3.4/site-packages/ipdbplugin.py", line 62, in debug
    traceback.print_exception(*err)
  File "/usr/lib/python3.4/traceback.py", line 169, in print_exception
    for line in _format_exception_iter(etype, value, tb, limit, chain):
  File "/usr/lib/python3.4/traceback.py", line 146, in _format_exception_iter
    for value, tb in values:
  File "/usr/lib/python3.4/traceback.py", line 125, in _iter_chain
    context = exc.__context__
AttributeError: 'str' object has no attribute '__context__'
```

**Comment by hwkns** [comment]
The second argument to `traceback.print_exception()` must be an exception object, but apparently a `str` object is being used instead.  Here are a few related links:

https://github.com/ionelmc/nose-htmloutput/issues/1
https://github.com/nose-devs/nose/issues/725
http://bugs.python.org/issue17413

**Comment by hwkns** [comment]
@flavioamieiro @berinhard any thoughts?

**Comment by jseabold** [comment]
I also keep running in to this.

**Comment by flavioamieiro** [comment]
Thank you very much for reporting this @hwkns. I am truly sorry about how long it took to fix this, and I won't make excuses. I've fixed this in c3b7d39 and uploaded a new version (1.4.2) to pypi.

Again, I'm sorry for taking so long.

/cc @jseabold

**Comment by hwkns** [comment]
<3 <3 <3
